core/src/main/resources/jenkins/model/ProjectNamingStrategy/PatternProjectNamingStrategy/config.groovy

```diff
@@ -4,7 +4,7 @@ package jenkins.model.ProjectNamingStrategy.PatternProjectNamingStrategy
 def f=namespace(lib.FormTagLib)
 
 f.entry(title:_("namePattern"), field:"namePattern") {
-    f.textbox(value:h.defaulted(instance?.namePattern, descriptor.DEFAULT_PATTERN),class:"fixed-width")
+    f.textbox(value: h.defaulted(instance?.namePattern, descriptor.DEFAULT_PATTERN), clazz: "fixed-width")
```

Comment:
just remove imo, would be more consistent
```suggestion
    f.textbox(value: h.defaulted(instance?.namePattern, descriptor.DEFAULT_PATTERN))
```

Reply:
Not a lot of regex fields in core to be consistent with 🤷  We probably should style https://github.com/jenkinsci/jenkins/blob/e681b4ff928435a500dcfb372cc9fc15b1c94413/core/src/main/resources/hudson/model/ListView/configure-entries.jelly#L56 similarly.

Reply:
I don’t think we should be styling the input any different to a standard one. Why is it special?

Reply:
It's a regular expression in both cases, which is code, which is typically displayed this way.

Why are shell scripts special?

https://github.com/jenkinsci/jenkins/blob/c73ad5d922a0d02101ca392b85b5dc358b766e98/core/src/main/resources/hudson/tasks/Shell/config.groovy#L28
https://github.com/jenkinsci/jenkins/blob/c73ad5d922a0d02101ca392b85b5dc358b766e98/core/src/main/resources/hudson/tasks/BatchFile/config.jelly#L29

Reply:
She’ll scripts get syntax highlighting...

not blocking this anyway, it seems unneeded to style it this way imo but I don’t really want to debate it any more